raw_data/preprocessed/africa/mauritania_french.txt

```diff
@@ -0,0 +1,251 @@
+
+(85) Le procès-verbal fera mention des objets saisis et le cas échéant, de la constitution d'un gardien de saisie.
+
+(86) Sans préjudice des prérogatives reconnues au ministère public, le Ministre chargé de l'Environnement ou son représentant sont chargés, dans l'intérêt général, de la poursuite des infractions commises en matière d'atteinte à l'environnement.
+
+(87) Sans préjudice des dispositions de l'article 86 ci-dessus, l'action publique peut être mise en mouvement par les associations de l'environnement, ou les collectivités locales.
+
+(88) La recherche et la constatation de l'infraction, la saisie des moyens de preuve dans les habitations et leurs annexes ne peuvent avoir lieu que dans les formes prescrites par le code de procédure pénale.
+
+(89) Seront passibles d'une amende de 3.000 ouguiyas à 200.000 ouguiyas les personnes physiques ou morales qui auront: [1] détenu ou abandonné des déchets en infraction aux dispositions des articles 60 et 61; [2] effectué des rejets en infraction aux dispositions des articles 45 et 49 ci-dessus.
+
+(90) Seront passibles d'une amende de 5.000 ouguiyas à 500.000 ouguiyas ceux qui auront: [1] négligé de remettre des déchets qu'ils produisent ou détiennent en violation des prescriptions du cahier des charges visés à l'article 66; [2] effectué des rejets interdits ou effectué sans autorisation des rejets soumis à autorisation en application de l'article 39 ci-dessus ou contrevenu aux conditions de l'autorisation dont ils étaient titulaires; [3] effectué des prises d'eau, mis en place sur le domaine public des aménagements, appareils au installations, ou creusé un puits en vue d'effectuer des prises d'eau sans l'autorisation réglementaire; [4] ouvert, implanté ou agrandi, accru la capacité de production, modifié substantiellement les caractéristiques techniques d'une installation portée sur la nomenclature prévue à l'article 52 ci-dessus ou auront commencé des travaux à ces effets sans l'autorisation requise, auront méconnu les règlements applicables à leurs installations ou les prescriptions de l'autorisation dont ils étaient titulaires; [5] introduit ou tenté d'introduire en Mauritanie des espèces animales ou végétales visées à l'article 28; [6] enfreint les dispositions de l'article 74 et 75 de la présente Loi; [7] se sont rendus coupables d'une dégradation de l’esthétique environnemental au sens de l'article 76 de la présente Loi.
```

Comment:
The last word in the article "Loi" is capitalized. This is consistent with the pdf but I'm just wondering why "loi" is capitalized here.

Reply:
I'm not sure. You are correct that these are the only two instances of `présente Loi`. I am not too concerned about inappropriate capitalization because the application converts all text to lowercase before comparison. Nonetheless, I will replace these two instances with `présente loi` for consistency.